en/docs/administer/configure-developer-portal-self-sign-up.md

```diff
@@ -101,4 +104,47 @@ Once you complete these steps, the Choreo support team will perform the final se
  
 Users can click **LOGIN/SIGN UP** and then click **Create an account** to sign up to your Developer Portal without having to send you a request to register them.
 
-![Create an account](../assets/img/administration/self-sign-up/create-an-account.png){.cInlineImage-half}        
+![Create an account](../assets/img/administration/self-sign-up/create-an-account.png){.cInlineImage-half} 
+
+## Approve user accounts
+
+To check the users who try to access your Developer Portal via self-sign-up, you can follow either of the following approaches:
+
+- Enable auto-approval so that every user who signs up is allowed to access your Developer Portal by default.
+- Manually check each sign-up and approval or reject it as required.
+
+### Enable auto-approval
+
+To enable auto-approval, follow the steps given below:
+
+1. Click **Settings** in the left navigation menu of the Choreo Console.
+
+2. Click **Self Signups**. 
+
+3. Toggle the **Auto-approval** switch.  This enables your account to automatically approve user accounts that create accounts to access your Developer Portal.
+
+    ![Auto-approval](../assets/img/administration/self-sign-up/auto-approval.png){.cInlineImage-threeQuarters}
+
+Once you enable auto-approval, the users can sign in to your Developer Portal and view your APIs and applications soon after they create an account.
+
+### Manually approve/reject user accounts
+
+If you have not enabled auto-approval, you need to follow the procedure given below to manually approve/reject the sign-up:
+
+1. Once the user creates an account, Choreo sends an email to ask the user to confirm the account. This email is similar to the one in the image below.
+
+    ![Confirm the account](../assets/img/administration/self-sign-up/confirm-account.png){.cInlineImage-half}
+
+2. The user needs to click **Confirm Account**.
+
+3. Click **Settings** in the left navigation menu of the Choreo Console, and then click **Self Signups**. You will see the user account listed for approval as shown below:
+
+    ![Pending approval](../assets/img/administration/self-sign-up/approval-pending.png){.cInlineImage-threeQuarters}
+
+4. If you want to approve the user account, click **Approve**. If not, click **Reject**.
```

Comment:
From the account list, to approve a specific user account, click **Approve** for that user account. If you wish to reject, click **Reject**.

Reply:
Can you add a reason as to why it was rejected?